Update maker-apps and Helium Hotspot App pages

waveform06 opened the pull request:
Added section for hotspotty app and hotspot utility app updated longap to recommend using Linxdot app

waveform06 committed:
Update maker-apps.mdx

Added section for hotspotty app and hotspot utility app
updated longap to recommend using Linxdot app

Sophi reviewed (commented):
_The [Helium Hotspot App](/wallets/app-wallet) no longer supports changes on hotspots other than the Helium Branded hotspot._ - suggest
The [Helium Hotspot App](/wallets/app-wallet) no longer supports maker hotspots other than the Helium Branded hotspot (sometimes called called the OG).

samgutentag committed:
formatting updates, sections dividers, capitalization

samgutentag commented:
> _The [Helium Hotspot App](/wallets/app-wallet) no longer supports changes on hotspots other than the Helium Branded hotspot._ - suggest The [Helium Hotspot App](/wallets/app-wallet) no longer supports maker hotspots other than the Helium Branded hotspot (sometimes called called the OG).

added this in [3170a84](https://github.com/helium/docs/pull/1072/commits/3170a8402b9fb14ae0943244965e2ebf6493149d)

samgutentag committed:
moving helium Hotspot app to different section

- moving helium Hotspot app page under “MIne HNT”
- redirects for page move
- sidebar updates for page move
- update sidebar label
- update page title
- hide page contents one layer deep, add tabs for tabs

samgutentag committed:
move sunset information to Hotspot app page
